common/djangoapps/third_party_auth/saml_configuration/tests/test_saml_configuration.py

```diff
@@ -45,7 +46,7 @@
 TEST_PASSWORD = 'testpwd'
 
 
-@unittest.skipUnless(testutil.AUTH_FEATURE_ENABLED, testutil.AUTH_FEATURES_KEY + ' not enabled')
```

Comment:
just a pref thing, but what do you think we rename `skip_tpa_tests` to something more explanatory like `skip_thirdparty_tests` or `skip_unless_thirdparty` or any other variation to clarify the intent? The intended skip would read like `skip_if_thirdparty_auth_app_is_not_installed` but that is way too verbose .. as long we somehow clarify `tpa` I think we will be in good shape

Reply:
yeah I didn't really want a super long name to have to type, but I guess `skip_unless_thirdpartyauth` could work. Third party is probably explanatory enough here, but makes me think of just general external django apps.

Reply:
yup like that name